spec/index.bs

```diff
@@ -5393,18 +5403,27 @@ dictionary GPUComputePassDescriptor : GPUObjectDescriptorBase {
 
     : <dfn>writeTimestamp(querySet, queryIndex)</dfn>
     ::
+        Writes a timestamp value into |querySet| when all previous commands have completed executing.
 
         <div algorithm="GPUComputePassEncoder.writeTimestamp">
             **Called on:** {{GPUComputePassEncoder}} this.
 
             **Arguments:**
             <pre class=argumentdef for="GPUComputePassEncoder/writeTimestamp(querySet, queryIndex)">
-                querySet:
-                queryIndex:
+                querySet: The query set that will store the timestamp values.
+                queryIndex: The index of the query in the query set.
             </pre>
 
             **Returns:** {{undefined}}
 
+            If any of the following conditions are unsatisfied, generate a validation error and stop.
+            <div class=validusage>
+                - |querySet| is [$valid to use with$] |this|.
```

Comment:
Maybe the redundant validation could be factored?

Same comment as for the other writeTimestamp validation.

Reply:
I think the entire definition will be the same between the two (except the argumentdef table will link to different places). Let's dedup as much as possible.

Reply:
Agree with Kai, they are different arguments validation on different objects (query set on encoders, query index on query set and query index on encoders), it's no easy to define them at same entrance.